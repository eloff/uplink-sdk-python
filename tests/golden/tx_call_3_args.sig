r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 89587958233495345488327815066712159206143705278919979327767424079371737221769
