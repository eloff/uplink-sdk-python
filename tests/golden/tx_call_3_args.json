{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }, 
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }, 
                    {
                        "contents": 1, 
                        "tag": "VInt"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE04OTU4Nzk1ODIzMzQ5NTM0NTQ4ODMyNzgxNTA2NjcxMjE1OTIwNjE0MzcwNTI3ODkxOTk3OTMyNzc2NzQyNDA3OTM3MTczNzIyMTc2OQ=="
}